packages/grafana-ui/package.json

```diff
@@ -66,7 +66,7 @@
     "react-table": "7.0.0",
     "react-transition-group": "4.3.0",
     "slate": "0.47.8",
-    "storybook-dark-mode": "0.4.0",
+    "storybook-dark-mode": "0.6.1",
```

Comment:
I think this should be in the `devDependencies`.

Reply:
@Clarity-89 Thanks for the review!

About this, initially I was going toward same direction as yours, however something changed my mind.

If no code is using, I would conclude same.
But I found [one code is using this this library](https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/utils/storybook/withTheme.tsx#L6). And I think there should be just one package listed in `package.json`, not more. Thus, I decided on `dependencies`.

But since this kind of case is also new to me, I would like to learn if there would be any better way to deal with this. What do you think?

Reply:
> And I think there should be just one package listed in package.json, not more. 

Yeah, it was included by accident in both dev and normal dependencies, and all other storybook-related addons are listed in `devDependencies`. So let's keep this addon in the `devDependencies` as well.

@tskarhed do we also need to update `yarn.lock`? What's the process for that?